devel/ortp/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        BelledonneCommunications ortp 5.0.18
+github.tarball_from archive
+revision            0
+
+categories          devel
+platforms           darwin
+license             GPL-3
+maintainers         {outlook.de:judaew @judaew} openmaintainer
+
+description         A Real-time Transport Protocol (RTP) library
+long_description    \
+    oRTP is a C library implementing the RTP protocol (rfc3550). It is \
+    available for most unix clones (primilarly Linux and HP-UX), and \
+    Microsoft Windows.
+
+checksums           rmd160 a59b92e1c08ed711eaf0456ea0d9dac9eaef24dd \
+                    sha256 a0039438970100c7e7baf19825ffdc8407f987f7659a5c76be5f4c93be9b5746 \
+                    size 234103
+
+depends_lib         port:bctoolbox
+
+patchfiles          patch-fix-Frameworks-path.diff
+
+configure.args-append \
+                    -DENABLE_STATIC=NO \
+                    -DENABLE_DOC=NO
+
+post-destroot {
+    move ${destroot}${prefix}/share/doc/ortp-. \
+        ${destroot}${prefix}/share/doc/ortp
+}
+
+variant docs description {Build and install documentation in HTML format} {
+    depends_build-append \
+                    port:doxygen
+
+    configure.args-append \
+                    -DENABLE_DOC=YES
```

Comment:
it would be cleaner here to do `configure.args-replace -DENABLE_DOC=NO -DENABLE_DOC=YES` instead of appending the new argument

Reply:
This is a great suggestion, thank you!